docs/Getting-Started.md

```diff
@@ -0,0 +1,270 @@
+    npm install --save reactify
+    npm install --save watchify
+
+To use these modules in development, we recommend adding the following lines to the `scripts` key of your `package.json` file to set up automatic JSX compiling and configure testing:
+
+    "scripts": {
+      "start": "watchify -d [YOUR MAIN.JS OR SIMILAR FILE] -o [YOUR OUTPUT BUNDLE.JS] -v",
+      "build": "NODE_ENV=production browserify [YOUR MAIN.JS OR SIMILAR FILE] | uglifyjs -cm > [YOUR OUTPUT BUNDLE.JS]"
+    }
+
+Include this `browserify` transform in your `package.json` file as well:
+
+    "browserify": {
+      "transform": [
+        [
+          "reactify",
+          {
+            "es6": true
+          }
+        ],
+        "envify"
+      ]
+    }
+
+The `browserify` transform will cause `watchify` to use `reactify` to compile your JSX and `es6` syntax like the `Spread` syntax, and allow you to use `envify` for node environment variables in the browser.
+
+After adding this code, run
+
+    npm start
+
+during development to automatically compile all JSX and Javascript syntax into one `bundle.js` which you can then directly link to your `index.html` file. Use
+
+    npm run build
+
+to compile a production ready bundle of your code.
+
+## TodoCreateForm.jsx
+Let's start by creating the top level Todo component using `Tux's` `createOwnerClass`. The Owner class is also responsible for the state of our Todo application. In `Tux` you are able to easily connect your Owner components to their stores using the `connectOwnerToStore` method which allows you to designate static properties and methods you want to automatically expose child components on the key of `nearestOwnerProps`. As the top-level Todo component it is also be responsible for exposing its methods and properties its children components. This is accomplished by using the `registerOwnerProps` method, which allows you to specify which properties and methods you want to expose to children components. Note these properties and methods should be static, if you need to pass down mutable properties, use the standard `React` practices to flow down `props`.
+
+**NOTE** We are going to build out the `todoStore`, `todoActions`, and the `TodoCreateForm` later in the Getting Started Guide.
+
+```javascript
+    var React = require('tux/React');
+    var todoActions = require('./todoActions');
+    var todoStore = require('./todoStore');
+    var Todo = require('./Todo.jsx');
+    var TodoCreateForm = require('./TodoCreateForm.jsx');
+
+    var TodoViewOwner = React.createOwnerClass({
+      getInitialState: function () {
+        return {
+          todos: todoStore.getAll() // Will get all of the Todos that are stored in the TodoStore
+        };
+      },
+
+      // This method connects this owner component to the specified store. Additionally it also adds and removes the change listeners from the store during the appropriate lifecycle events, specifically componentDidMount and componentWillUnmount.
```

Comment:
Maybe should specify that this is also invoked with the context of the component.